rolling/ci-clang-tidy.yaml

```diff
@@ -0,0 +1,89 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
```

Comment:
> Remove acceptance of connext license from build which does not use it.

Looks like you dropped this intentionally. How does this build not use Connext? What am I missing?

Reply:
You're not missing anything. I thought that I had already excluded rmw_connextdds from the repos file this build targets but I have not. I'll add another PR to address that in the repos file.

Do you think it is prudent to add a package selection argument ignoring rmw_connextdds as added protection in case the repos file changes in the future or is removing it from the repos file enough?

Reply:
> Do you think it is prudent to add a package selection argument ignoring rmw_connextdds as added protection...?

Nah, I think it would be misleading, personally. Dropping it from the `.repos` file seems sufficient for me.